### ISSUE TITLE
Frequency table search problem

### ISSUE THREAD

**Issue by T-a-c-h-y-o-n** [man-group/dtale#837]
Hi,
In the new version, when I enter a value from the keyboard for the search, the d-tale table disappears completely. I wanted to try it in colab, but d-tale doesn't work at all in colab.
[ss.webm](https://github.com/man-group/dtale/assets/40146917/24edc43b-9aae-4bde-ac70-b974324c5f61)

**Comment by T-a-c-h-y-o-n** [comment]
The error in question is valid for numerical variables. Maybe it was there in previous versions, but I'm just noticing it.

**Comment by aschonfeld** [comment]
@T-a-c-h-y-o-n here's a workaround for using D-Tale in colab until I can publish a new version (notice the line for `ACTIVE_HOST`):
```
import pandas as pd
import dtale
import dtale.app as dtale_app

dtale_app.USE_COLAB = True
dtale_app.ACTIVE_HOST = '0.0.0.0'
dtale.show(pd.DataFrame(dict(x=[1,2,3], y=[2,4,8]))
```

I'm going to start digging into that search bug...

**Comment by aschonfeld** [comment]
@T-a-c-h-y-o-n  just released v3.10.0 to pypi (should be on conda-forge soon) with this fix included.
